string camera_name
---
bool success
string message
